app/controllers/hosts_controller.rb

```diff
@@ -713,7 +713,7 @@ def find_multiple
     # Lets search by name or id and make sure one of them exists first
     if params.key?(:host_names) || params.key?(:host_ids) || multiple_with_filter?
       @hosts = resource_base.search_for(params[:search]) if multiple_with_filter?
-      @hosts ||= resource_base.where("hosts.id IN (?) or hosts.name IN (?)", params[:host_ids], params[:host_names])
+      @hosts ||= resource_base.merge(Host.where(id: params[:host_ids]).or(Host.where(name: params[:host_names])))
```

Comment:
This was likely one of the places that failed previously in katello, in order to get the correct operator precedence for the `OR` clause we need to merge the scopes.